lib/iris/cube.py

```diff
@@ -2785,6 +2785,34 @@ def slices(self, ref_to_slice, ordered=True):
 
         return _SliceIterator(self, dims_index, dim_to_slice, ordered)
 
+    def invert_dims(self, dimensions):
+        """
+        Invert the specified dimensions of a cube in-place.
+
+        Args:
+
+        * dimensions (iterable of ints):
+            Determines which dimensions will be inverted.
+
+        """
+        dimensions = tuple(set([int(dim) for dim in dimensions]))
+
+        for dim in dimensions:
+            coord = self.coord(contains_dimension=dim)
+            coord_dims = self.coord_dims(coord)
+            if len(coord_dims) > 1:
+                msg = ('Currently multidimensional coordinate inversion is '
+                       'not supported')
+                raise RuntimeError(msg)
```

Comment:
Perhaps `CoordinateMultiDimError` would be better? (https://github.com/SciTools/iris/blob/master/lib/iris/exceptions.py#L48)